templates/sshd_config.erb

```diff
@@ -254,3 +254,13 @@ Match <%= key %>
 <%     end -%>
 <%   end -%>
 <% end -%>
+
```

Comment:
These spaces are causing the issue since it doesn't match the fixture file. Put your opening `if` just after the last `end` and then put an empty line. Same with the if below it.